pandas/tests/extension/conftest.py

```diff
@@ -31,11 +31,11 @@ def all_data(request, data, data_missing):
 
 
 @pytest.fixture
-def data_repeated():
+def data_repeated(data):
     """Return different versions of data for count times"""
     def gen(count):
         for _ in range(count):
-            yield NotImplementedError
+            yield data
```

Comment:
Previously, you could end up with a `Series([NotImplementedError] * count])` if you didn't override this.

Reply:
Is this specific this to this PR, or could it go in a logically-independent PR?

Reply:
Independent. Will split.